src/app/cube/details/components/action-panel/editorial-action-pad/editorial-action-pad.component.ts

```diff
@@ -82,7 +84,7 @@ export class EditorialActionPadComponent implements OnInit {
   async createRevision() {
     this.closeRevisionModal();
     this.toaster.success('One Moment Please', 'Your revision is being created.');
-    await this.learningObjectService
+    await this.revisionsService
       .createRevision(this.learningObject.cuid).then(async (revisionUri: any) => {
         this.revisedLearningObject = (await this.learningObjectServiceUri.fetchUri(revisionUri.revisionUri).toPromise())[0];
```

Comment:
Can we add an inline comment TODO here to let us know to update the return value of createRevision to be the revised learning object rather then a URI that we just make a GET request directly after. (Essentially we are waisting time and requests by making a duplicate request that can be simplified by just returning the response that the client needs).